jaeger/charts/jaeger/values.yaml

```diff
@@ -0,0 +1,15 @@
+# Default values for tracing.
+
+namespace: linkerd-jaeger
+
+collector:
+  image: 
+    name: omnition/opencensus-collector
+    version: 0.1.11
+    pullPolicy: Always
+  # resources:
+jaeger:
+  image: 
+    name: jaegertracing/all-in-one
+    version: 1.19.2
+    pullPolicy: Always
```

Comment:
let's try to get newlines at the end of all files unless there's a reason not to